src/content/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package.mdx

```diff
@@ -112,3 +112,45 @@ To update the on-host integration using Zypper:
    sudo zypper update <var>INTEGRATION_FILE_NAME</var> -y
    ```
 2. [Restart](/docs/infrastructure/new-relic-infrastructure/configuration/start-stop-restart-check-infrastructure-agent-status#linux) the infrastructure agent.
+
+## Update Microsoft Windows integrations [#windows]
+
+Here are two approaches, depending on the integration you're using.
+
+### Microsoft Windows services integration [#ms-services]
+
+The Microsoft Windows services integration is bundled with the New Relic infrastructure agent, so to update the integration, you need to upgrade the infrastructure agent. See [Update the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/update-or-uninstall/update-infrastructure-agent).
+
+### Microsoft SQL Server monitoring integration [#ms-sqlserver]
```

Comment:
There are [multiple integrations](http://nr-downloads-ohai-staging.s3-website-us-east-1.amazonaws.com/infrastructure_agent/windows/integrations/) for windows. 

We(coreint) support:
```
nri-apache/                                                                      -
nri-cassandra/                                                                   -
nri-consul/                                                                      -
nri-couchbase/                                                                   -
nri-elasticsearch/                                                               -
nri-f5/                                                                          -
nri-haproxy/                                                                     -
nri-kafka/                                                                       -
nri-memcached/                                                                   -
nri-mongodb/                                                                     -
nri-mssql/                                                                       -
nri-mysql/                                                                       -
nri-nagios/                                                                      -
nri-nginx/                                                                       -
nri-postgresql/                                                                  -
nri-rabbitmq/                                                                    -
nri-redis/                                                                       -
nri-varnish/                                                                     -
nri-vsphere/                                                                     -
```

Caos supports, however the upgrade process should be exactly the same
```
nri-jmx/                                                                         -
nri-snmp/                                                                        -
nrjmx/    
```

I am not sure about `nri-winpkg`